feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/SearchScreenViewModel.kt

```diff
@@ -47,35 +47,44 @@ class SearchScreenViewModel @Inject constructor(
             initialValue = Timetable(),
         )
 
-    private val searchFilterUiState: MutableStateFlow<SearchFilterUiState> = MutableStateFlow(
-        SearchFilterUiState(),
+    private val filterStateFlow: MutableStateFlow<Filters> = MutableStateFlow(
+        Filters(),
     )
 
     val uiState: StateFlow<SearchScreenUiState> = buildUiState(
         searchQuery,
-        searchFilterUiState,
+        filterStateFlow,
         sessionsStateFlow,
-    ) { searchQuery, searchFilterUiState, sessions ->
+    ) { searchQuery, filters, sessions ->
         val searchedSessions = sessions.filtered(
             Filters(
-                days = searchFilterUiState.selectedDays,
-                categories = searchFilterUiState.selectedCategories,
-                sessionTypes = searchFilterUiState.selectedSessionTypes,
-                languages = searchFilterUiState.selectedLanguages,
                 searchWord = searchQuery,
+                days = filters.days,
+                categories = filters.categories,
+                sessionTypes = filters.sessionTypes,
+                languages = filters.languages,
             ),
```

Comment:
It might be better to implement like `filters.copy(searchWord = searchQuery)` for sake of simplicity, but I wondered that it might violate Single Source of Truth principle 🤔

Reply:
I don't want to copy UiState for creating consistent UI but filter is not UiState so I think it is ok.